### PR TITLE
chore: Minor refactor

### PR DESCRIPTION
**Pull request by jamesgeorge007**
refactor code to be more concise.

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @shadowspawn

**Comment by shadowspawn** [comment]
I noticed the typo, and am pleased the tests picked up the error!

The repetition in the existing code does annoy me a little, but the code is simple and easy to understand.  Concise is not always better. On this occasion, I am going to say no thanks.

Thank you  for your contributions.

File: index.js [hunk 1]
```diff
@@ -544,17 +544,16 @@ Command.prototype.executeSubCommand = function(argv, args, unknown, executableFi
...
-    bin = localBin + '.ts';
-    isExplicitJS = true;
-  } else if (exists(localBin + '.mjs')) {
-    bin = localBin + '.mjs';
-    isExplicitJS = true;
-  } else if (exists(localBin)) {
+  if (exists(localBin)) {
     bin = localBin;
+  } else {
+    ['js', 'ts', 'mjs'].some(function(extension) {
```
Comment (shadowspawn) [inline index.js]: Missing the dots on the file extensions.

Comment (jamesgeorge007) [reply index.js]: should I reopen with after having the respective changes :question:

Comment (shadowspawn) [reply index.js]: No thanks. (I used the review as a quick way of highlighting the broken line of code.)

**Pull request closed without merging**